include/SFML/Audio/InputSoundFile.hpp

```diff
@@ -63,8 +63,9 @@ class SFML_AUDIO_API InputSoundFile : NonCopyable
     ////////////////////////////////////////////////////////////
     /// \brief Open a sound file from the disk for reading
     ///
-    /// The supported audio formats are: WAV (PCM only), OGG/Vorbis, FLAC.
+    /// The supported audio formats are: WAV (PCM only), OGG/Vorbis, FLAC, MP3.
     /// The supported sample sizes for FLAC and WAV are 8, 16, 24 and 32 bit.
+    /// For MP3 files with big (>16kb) APEv2 tag there is a low chance of garbage decoded at the end.
```

Comment:
This comment should be clarified. What is "garbage", why does it happen, how to avoid?

Reply:
Can it be link to the issue comment? Added one with explanations.
Usually no need to avoid, chance is very low, just warning that APEv2 may not be removed and treated as mp3 data.

Reply:
Ideally the documentation should be self-contained, also for people working offline. If the description is a few sentences, can you copy it into the doc comment?

Reply:
https://github.com/SFML/SFML/issues/1232#issuecomment-641633168

This comment. OK, I try describe all in place.

Reply:
Updated the PR with full explanation.